### ISSUE TITLE
`ckeditor5-package-generator test`: Exit code for CI (document example CI workflow?)

### ISSUE THREAD

**Issue by tony** [ckeditor/ckeditor5-package-generator#124]
Thank you for this package, it is excellent!

## 📝 Provide a description of the new feature

As of 1.0.0-beta.5, `yarn test` will always pass in GitHub workflows / shell, even if there's errors

re: [`ckeditor5-package-tools test`](https://github.com/ckeditor/ckeditor5-package-generator/blob/e6689f662e9034c31281af367e7361f09cfebe60/packages/ckeditor5-package-generator/lib/templates/common/README.md#test)

My workaround is to pipe to `exit` or collect the exit code:

```yaml
- name: test
  run: xvfb-run yarn test --coverage --source-map --verbose | exit 1
```

Long form:

```yaml
- name: test
  run: |
      xvfb-run yarn test --coverage --source-map --verbose
      EXIT_CODE=$?
      exit ${EXIT_CODE}
```

Is this intended? If so should we document? (or maybe there's a more efficient way to make the exit code available?)

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by pomek** [comment]
It's indeed a bug. When the testing command fails (for any reason), it should end with the non-zero exit code.

**Comment by przemyslaw-zan** [comment]
I've experimented with our CI, and I can't reproduce the problem. I've tried:

- Adding failing test to the generator itself
- Adding failing test in the generated package within the generator
- Adding failing test in the generated package as a standalone repository

[Here are respective commits of each case,](https://github.com/ckeditor/ckeditor5-package-generator/compare/i/124) and as expected, all of them result in a failed CI check. I should also mention that generated packages do not contain `travis.yml`, it was added in the last case for the purpose of the test.

So what is the issue? In all experiments I ran, `yarn run test` returns exit codes correctly. If you're using GitHub workflows in package that you've generated, then there is nothing we can do about it, as we have not created config for that.
